packages/office-ui-fabric-react/src/components/DetailsList/DetailsList.types.ts

```diff
@@ -245,6 +245,11 @@ export interface IDetailsListProps extends React.Props<DetailsList>, IWithViewpo
    * Whether or not the selection zone should enter modal state on touch.
    */
   enterModalSelectionOnTouch?: boolean;
+
+  /**
+   * Rerender DetailsRow only when props changed. Might cause regression when depending on external updates.
```

Comment:
Would be good to document the default value via `@default`. This also makes it into the autodoc.